src/app/public/modules/file-attachment/file-attachment.component.ts

```diff
@@ -89,24 +92,33 @@ export class SkyFileAttachmentComponent implements AfterViewInit, AfterContentIn
     return this._disabled;
```

Comment:
The `disabled` input should specify a default of `false`.

And we also need to delete `This property accepts boolean values.` because `docs-tools` handles that now so that we don't have to call it out in the description.

Reply:
Done

Reply:
I'm not actually seeing this change. When I follow the PR preview link, we don't have the default and still have the unnecessary sentence in the property description. Is something wonky going on here? It looks like all the checks have passed.

Reply:
I hadn't pushed the change. I just did so you should have a new link soon.